nilearn/decomposition/base.py

```diff
@@ -345,7 +345,7 @@ def __init__(self, n_components=20,
         self.verbose = verbose
 
     def fit(self, imgs, y=None, confounds=None):
-        """Base fit for decomposition estimators : compute the embedded masker
+        """Compute the mask and the components across subjects
```

Comment:
Maybe "components" -> "PCA componets"

Reply:
I would not because this fit() is common to all estimators.